test/common/test-functions.js

```diff
@@ -51,6 +51,14 @@ function ph_set_val (sel, val) {
     el.dispatchEvent(ev);
 }
 
+function ph_set_text_direction(sel, direction) {
```

Comment:
I was going to complain that this function actually doesn't do enough to justify its existence. Sorry for the suggestion then, it looked a lot more involved originally  (with DOM searching and multiple commands). But indeed you actually don't call this anywhere, so this is just dead weight. Please remove -- this is fine in a follow-up if you want to land this quickly (due to blocking pixel tests).